### PR TITLE
Add Email validation

### PR DESCRIPTION
**Pull request by betses**
# Feature added

- Added email input validation using JavaScript
- Fix linter error

Please kindly review my code.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,17 +142,20 @@ <h2>Skills</h2>
...
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
           </h2>
         </div>
-        <form action="https://formspree.io/f/xoqrojkw" method="POST">
+        <form id="contact-me" action="https://formspree.io/f/xoqrojkw" method="POST">
           <div class="input-field">
-            <input type="text" name="name" maxlength="30" class="text-input" placeholder="Full name" required>
-            <input type="email" name="Email" class="text-input" placeholder="Email address" required>
+            <input type="text" name="name" id="user-name" maxlength="30" class="text-input" placeholder="Full name" required>
+            <input type="email" id="user-email" name="Email" class="text-input" placeholder="Email address" required>
```
Comment (Kossi-stack) [inline index.html:155]: - [ ] Kindly fix this
-  If a user submits the contact form with the email field containing uppercase characters an error message should show up.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Hello Microverse Project PR or Hello Microverse Project".
 - [ ] Please make sure to use [this readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and please be sure to edit all of the sections to better suit your project.
- [ ] Kindly make sure to use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse and make sure they are set up correctly and they all pass with green checkmarks.
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 - [ ] Kindly capitalize the first letter of your commit message in the next milestone so it looks professional.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (betses) [reply index.html:155]: I have already added the error message when the email field has uppercase characters on line 156. can you kindly please check the code.

File: index.html [hunk 2]
```diff
@@ -142,17 +142,20 @@ <h2>Skills</h2>
...
+            <div class="error-message">
```
Comment (topeogunleye) [inline index.html:156]: - [ ] Great job adding your error message for "Emails written in Capital Letter", however pls kindly add your error message close to your submit button and not above the textarea like you have done.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @betses,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (betses) [reply index.html]: Thanks for the feedback I have edited my error message.

**Review by danmainah — APPROVED** [review]
Hi @betses and @betse-1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**